README.md

```diff
@@ -291,7 +291,7 @@ Table of Contents
   * [wistia.com](https://wistia.com/) — Video hosting with viewer analytics, HD video delivery and marketing tools to help understand your visitors, 25 videos and Wistia branded player
   * [yammer.com](https://www.yammer.com/) — Private social network standalone or for MS Office 365. Free with a bit less admin tools and user management features
   * [zoom.us](https://zoom.us/) — Secure Video and Web conferencing, add-ons available. Free limited to 40 minutes
-
+  * [shtab.app](https://shtab.app/) - Project management service that makes collaboration in the office and remotely transparent with tracker based on AI.
```

Comment:
while this looks good (my wife can speak russian) I can't really add all russian site here that I, and others, cant easily verify its free etc.

Do they have a english version?

Reply:
There is currently no english version of the app, but the team is working on it. Can i put draft on PR while team work on english version?

Reply:
yup, you can do that

Reply:
Hi! Now shtab.app have multiple languages at system (English and Russian)

Reply:
very good, thank you for coming back :)